dashboard/app/controllers/pd/regional_partner_contact_controller.rb

```diff
@@ -11,4 +8,11 @@ def new
       }.to_json
     }
   end
+
+  # Get /pd/regional_partner_contacts/:contact_id/thanks
+  def thanks
+    regional_partner_contact = Pd::RegionalPartnerContact.find(params[:contact_id])
```

Comment:
can (we use) cancan(can) to load this via `load_resource`? :p

Reply:
Yes we can(can(can))